### PR TITLE
refactor home page code ♻️

### PR DESCRIPTION
**Pull request by jibola2588**
## Related Issue

refactor home page code #723 

## Description

refactor home page code

## Screenshots

Screenshot (87)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
     "react-icons": "^4.8.0",
     "react-paginate": "^8.2.0",
     "react-router-dom": "^6.11.2",
-    "styled-components": "6.0.0-rc.2"
+    "styled-components": "6.0.0-rc.2",
+    "workspace-name": "^0.0.0"
```
Comment (priyankarpal) [inline package.json:38]: why these two dependencies?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
please add the issue number without the issue number PR might be closed.

File: package.json (hunk 1 above)
Comment (jibola2588) [reply package.json:38]: I fetched Upstream and realized a contributor added react-paginate, so I had to install the dependency in my local

**Comment by jibola2588** [comment]
I will add the issue number now

**Comment by jibola2588** [comment]
The issue number has been added

File: package.json (hunk 1 above)
Comment (priyankarpal) [reply package.json:38]: ok

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉 Now the code is easier to understand thanks for your awesome contribution. 👏🏻 ❤️

**Comment by jibola2588** [comment]
Thanks sir

**Pull request merged**